### ISSUE TITLE
New feature request Javascript logs

### ISSUE THREAD

**Issue by mudyc** [mattheworiordan/capybara-screenshot#160]
I think it makes a good improvement if capybara-screenshot could have javascript logs in addition to html page.

**Comment by mattheworiordan** [comment]
Agreed @mudyc. You posted some code to [Ably](https://www.ably.io) the other day, any chance you could get that into a PR?
